README.md

```diff
@@ -31,14 +29,6 @@ Please provide unit tests for your contributions. Run tests with gradle:
 Unit tests use a mock networking and storage stack so that different components can be tested in isolation. The unit tests should not connect to any external services.
 
 
-## Publish the library to maven central
```

Comment:
Publishing is now done from CI, no instructions are necessary.

Reply:
When is publishing done especifically? After merge to develop or at what phase?

Reply:
@danilo04 Sorry, I should have listed that in the PR description 🤦 

Here is the list of when artifacts will be published and what the version name will be:

* For all tags -> `{tag-name}`
* For all commits in `develop` (so PR merges) -> `develop-{sha1}`
* For all commits for open PRs - you can open a draft PR to get it to publish. -> `{prNumber}-{sha1}`

These settings can be tweaked, but this is the way we do it for all Android libraries.

---

Typical workflow is: 

* Open a PR and test the published version (which will be printed out in the Buildkite job and follows the naming pattern above)
* Merge the PR into `develop`
* Update `simplenote-android` PR to use the merge commit which is `develop-{sha1}`
* Merge the `simplenote-android` PR
* At the time of release, we create a new tag for the `simperium-android` and update the version value in `simplenote-android` to the tagged value.

Reply:
Thanks for the explanation. That makes it easier also for testing the changes with `simplenote-android`.

Reply:
I'll add composite build support which will make it even easier to test it. I just need a little bit of time to finish up some other tasks. 😅

Reply:
btw just wanted to take the occasion to thank @oguzkocer again on all his work on that, and especially the new way we use for publishing Android libraries. 😍 
With the auto-versioning following the `develop-{sha1}`/`{prNum}-{sha1}` patterns you mentioned above, which means that developers don't need to create "beta tags" on the libs anymore, this make testing lib changes way easier for everybody! ❤️ 

_(The only tricky part remaining is for the release manager to find the right "next version number" to use at release time i.e. to decide if they need to bump major vs minor vs patch, but usually minor bump makes sense for Android libs given we don't rely on SemVer in Gradle for those anyway, so that's not a big deal anyway)_

Reply:
Even if the manual instructions are not necessary anymore, do you think I'd still make sense to mention in the readme the auto-publishing by CI and the version name patterns we use (i.e. put your great explanations from [above](https://github.com/Simperium/simperium-android/pull/232/files#r690695360) in the README)?

Reply:
@AliSoftware As one of the developers, I would say yes, because a new developer can start working immediately with this information.